packages/wallet-sdk/src/connection/WalletLinkConnection.ts

```diff
@@ -246,15 +219,14 @@ export class WalletLinkConnection {
     this.diagnostic?.log(EVENTS.STARTED_CONNECTING, {
       sessionIdHash: Session.hash(this.sessionId),
     });
-    this.ws.connect().subscribe();
+    this.ws.connect();
   }
 
   /**
    * Terminate connection, and mark as destroyed. To reconnect, create a new
    * instance of WalletSDKConnection
    */
   public destroy(): void {
```

Comment:
We were calling `unsubscribe` before which would clear the previous listeners.  I don't see that happening anymore.  Is there a possibility that the listeners will now get called after the connection is destroyed, for example through a race condition?

Reply:
https://github.com/coinbase/coinbase-wallet-sdk/pull/1040